dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -107,26 +107,39 @@ entities. Only tracked entities with an enrollment into the `program` will be re
 
 ### `*.parameter.TrackedEntityRequestParams.enrollmentStatus`
 
-Get tracked entities with an enrollment in the given status.
+Get tracked entities that have at least one enrollment with the status specified.
+
+Valid options are:
+- `ACTIVE`
+- `COMPLETED`
+- `CANCELLED`
 
 ### `*.parameter.TrackedEntityRequestParams.programStatus`
 
-Get tracked entities with an enrollment in the given status.
+Get tracked entities that have at least one enrollment with the status specified.
+
+Valid options are:
+- `ACTIVE`
+- `COMPLETED`
+- `CANCELLED`
 
 **DEPRECATED as of 2.42:** Use parameter `enrollmentStatus` instead.
 
-See `enrollmentStatus` for details.
 
 ### `*.parameter.TrackedEntityRequestParams.followUp`
 
+Get tracked entities that has at least one enrollment that is marked with follow up.
+
 ### `*.parameter.TrackedEntityRequestParams.updatedAfter`
 
-Get tracked entities updated after given date and time.
+Get tracked entities that is updated after the given time. The update can be on the tracked entity
```

Comment:
This isn't very clear to me.
We should have `updatedAt` on a tracked entity always in sync with any `updatedAt` of enrollments, events or attributes. Here it seems that we are collecting all the `updatedAt` from the listed entities and providing the latest one, implying that the tracked entity one could not be the latest one

Reply:
Reworded it a bit. Lmk what you think.

Reply:
LGTM, I would add the relationships and add the same description for `updatedBefore`